java/api/src/main/java/org/ray/api/ActorCall.java

```diff
@@ -651,4 +651,37 @@ default void call(RayFuncVoid1<A> f) {
     Ray.internal().callActor(f, (RayActor) this, args);
   }
 
+  // ===========================
+  // Cross-language methods.
+  // ===========================
+  default RayObject call(String functionName) {
```

Comment:
I don't think it's a good idea to add Python function calls on `RayActor` interface since we already have a more concrete interface `RayPyActor` available.

Reply:
I moved Python actor call to PyActorCall.java, and RayPyActor extends PyActorCall.